label_namer.go

```diff
@@ -25,19 +25,30 @@ import (
 )
 
 // LabelNamer is a helper struct to build label names.
+// It translates OpenTelemetry Protocol (OTLP) attribute names to Prometheus-compliant label names.
+//
+// Example usage:
+//
+//	namer := LabelNamer{UTF8Allowed: false}
```

Comment:
I'd like to standardize always setting both UTF8Allowed as well as WithMetricSuffixes to be extra explicit, esp because most people will want to add suffixes until prometheus gets type/unit support

Reply:
Alright, they are all there I believe :)